doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1092 @@
+
+* :ref:`caf_sensor_data_aggregator`:
+
+  * Added unit tests for the library.
+
+* :ref:`caf_sensor_manager`:
+
+  * Updated to no longer use floats to calculate and determine if the sensor trigger is activated.
+    This is because the float uses more space.
+    Also, data sent to :c:struct:`sensor_event` uses :c:struct:`sensor_value` instead of float.
+
+Libraries for Zigbee
+--------------------
+
+* :ref:`lib_zigbee_application_utilities` library:
+
+  * Added :kconfig:option:`CONFIG_ZIGBEE_PANID_CONFLICT_RESOLUTION` for enabling automatic PAN ID conflict resolution.
+    This option is enabled by default.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`bt_fast_pair_provision_script`:
+
+  * Added a SHA-256 hash of the Fast Pair provisioning data to ensure its integrity.
+
+* :ref:`partition_manager`:
+
+  * Added:
+
+    * :kconfig:option:`CONFIG_PM_PARTITION_REGION_LITTLEFS_EXTERNAL`, :kconfig:option:`CONFIG_PM_PARTITION_REGION_SETTINGS_STORAGE_EXTERNAL`, and :kconfig:option:`CONFIG_PM_PARTITION_REGION_NVS_STORAGE_EXTERNAL` Kconfig options to specify that the relevant partition must be located in external flash memory.
+    * :kconfig:option:`CONFIG_PM_OVERRIDE_EXTERNAL_DRIVER_CHECK` to override the external driver check.
+      This is needed when using an external flash which is not using the :ref:`QSPI NOR <zephyr:dtbinding_nordic_qspi_nor>` driver from Zephyr.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``1d4404116a9a6b54d54ea9aa3dd2575286e666cd``, plus some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes specific to the |NCS|:
+
+* boot_serial:
+
+  * Upgraded from cddl-gen v0.1.0 to zcbor v0.4.0.
+  * Refactored and optimized the code, mainly in what affects the progressive erase implementation.
+  * Fixed a compilation issue with the echo command code.
+
+* imgtool: Added support for providing signature through a third party.
+* zephyr:
+
+  * Added initial support for leveraging the RAM-LOAD mode with the zephyr-rtos port.
+  * Added the MCUboot status callback support.
+    See :kconfig:option:`CONFIG_MCUBOOT_ACTION_HOOKS`.
+  * Edited includes to have the ``zephyr/`` prefix.
+  * Edited the DFU detection's GPIO-pin configuration to be done through DTS using the ``mcuboot-button0`` pin alias.
+  * Edited the LED usage to prefer DTS' ``mcuboot-led0`` alias over the ``bootloader-led0`` alias.
+  * Removed :c:func:`device_get_binding()` usage in favor of :c:func:`DEVICE_DT_GET()`.
+
```

Comment:
worth to mention: Made possible serial recovery of the image of nRF5340 cpunet even that simultaneous cores upgrade is disabled (CONFIG_NRF_MULTI_IMAGE_UPDATE=n ). Ref CONFIG_NRF53_RECOVERY_NETWORK_CORE.
this was added in https://github.com/nrfconnect/sdk-nrf/pull/8050

Reply:
You want me to mention this in the zephyr additions section? 
Also: we mention this new section a couple of lines further below under doc changes.